pkg/doc/verifiable/credential.go

```diff
@@ -667,10 +667,10 @@ func decodeIssuer(issuerBytes json.RawMessage) (Issuer, error) {
 	return issuer, err
 }
 
-// decodeCredentialSchemas decodes credential schema(s).
+// parseCredentialSchemas parses credential schema(s).
 //
 // credential schema can be defined as a single object or array of objects.
-func decodeCredentialSchemas(data *rawCredential) ([]TypedID, error) {
+func parseCredentialSchemas(data *rawCredential) ([]TypedID, error) {
```

Comment:
this function still seems to be a `decode`

Reply:
This function creates a new structure, that's why I used **parse** word here.

Reply:
parse refers to taking byte inputs and creating a structure.

Reply:
I see, thank you! Corrected.

Reply:
json.RawMessage is also bytes.

Reply:
Right! Corrected that as well.